srv/www/regionService/regionInfo.py

```diff
@@ -156,8 +156,12 @@ def create_smt_region_map(conf):
             try:
                 ipaddress.IPv6Network(ip_range)
             except ValueError:
-                msg = 'Could not proces range, improper format: %s'
-                logging.error(msg % ip_range)
+                if ip_range:
+                    # IPv6 is optional
+                    # so it is OK if not IPv6
+                    # present in config file
+                    msg = 'Could not process IPv6 range, improper format: %s'
+                    logging.error(msg % ip_range)
```

Comment:
If it's not an error I would log it at info or debug level.

Reply:
It is an error if there is a value but the operation `ipaddress.IPv6Network(ip_range)` failed

If it fails because there is no value in `ip_range` it is OK as IPv6 is not currently mandatory